config/routes/admin.rb

```diff
@@ -93,7 +93,12 @@
     end
   end
 
-  resources :rules, only: [:index, :new, :create, :edit, :update, :destroy]
+  resources :rules, only: [:index, :new, :create, :edit, :update, :destroy] do
+    member do
+      post :move_up
+      post :move_down
```

Comment:
I would rather use `patch` here, because this will update existing rules, whereas `post` is supposed to create something new. What do you think?

Reply:
I believe this would be inconsistent with every other action we have, and `PATCH` would be about updating the specific rule with provided params, which is different here.